core-services/prow/02_config/openshift/api/_pluginconfig.yaml

```diff
@@ -5,6 +5,7 @@ label:
       - xingxingxia
       - wangke19
       - gangwgr
+      - openshift-staff-engineers
```

Comment:
i think this needs to go under `allowed_teams`, e.g.:
https://github.com/openshift/release/blob/fd44743075db1228fb4930d1d7bd3037fffb3695/core-services/prow/02_config/_plugins.yaml#L2288-L2295

Reply:
@bparees Thanks for catching that. Done.